gcsio/src/test/java/com/google/cloud/hadoop/gcsio/GoogleCloudStorageFileSystemNewIntegrationTest.java

```diff
@@ -711,6 +711,42 @@ private void delete_directory(boolean parallelStatus) throws Exception {
     assertThat(gcsFs.exists(bucketUri.resolve(dirObject + "/d1"))).isFalse();
   }
 
+  @Test
```

Comment:
May you validate that this test will fail without your fix?

Reply:
Done, verified